api/pkg/db/model/model.go

```diff
@@ -80,6 +80,7 @@ type (
 		Description         string
 		URL                 string `gorm:"not null;default:null"`
 		DisplayName         string
+		Deprecated          string
```

Comment:
is it possible to use `bool` type here?

Reply:
yeah  it's possible,used string as while parsing we get it as string("true") from resources

Reply:
yeah but boolean in db will take less space and we won't any value other than T/F for now